test/common/network/cidr_range_test.cc

```diff
@@ -0,0 +1,240 @@
+#include "envoy/common/exception.h"
+
+#include "common/network/cidr_range.h"
+#include "common/network/address_impl.h"
+
+#include <sys/un.h>
+
+// We are adding things into the std namespace.
+// Note that this is technically undefined behavior!
+namespace std {
+
+// Pair
+template <typename First, typename Second>
+std::ostream& operator<<(std::ostream& out, const std::pair<First, Second>& p) {
```

Comment:
Sure, I thought this was only for implicit use but it's not as you point out, consider it resolved.